apps/test-app/package.json

```diff
@@ -16,7 +16,7 @@
     "@graphql-codegen/typescript-resolvers": "^2.6.1",
     "@graphql-codegen/typescript": "^2.4.8",
     "@types/node": "^16",
-    "functionless": "^0.27.1",
```

Comment:
is this intentional?

Reply:
it is not

Reply:
weird, that's not in my local

Reply:
Oh actually, it was intentional - it was causing a problem where differnet versions of functionless were installed from NPM and not fetched from the monorepo

Reply:
Fixed.